### PR TITLE
feat: apisix add extraVolumeMounts and extraVolume

### PR DESCRIPTION
**Pull request by anjia0532**
Signed-off-by: 赵安家 <anjia0532@gmail.com>

1. extraVolumes: []
2. extraVolumeMounts: []

### CODE REVIEW

File: charts/apisix/README.md [hunk 1]
```diff
@@ -76,6 +76,8 @@ The following tables lists the configurable parameters of the apisix chart and t
 | `apisix.luaModuleHook.configMapRef.name` | Name of the ConfigMap where the lua module codes store | "" |
 | `apisix.luaModuleHook.configMapRef.mounts[].key` | Name of the ConfigMap key, for setting the mapping relationship between ConfigMap key and the lua module code path. | `""` |
 | `apisix.luaModuleHook.configMapRef.mounts[].path` | Filepath of the plugin code, for setting the mapping relationship between ConfigMap key and the lua module code path. | `""` |
+| `extraVolumes` | Additional `volume`. | `[]` |
```
Comment (tokers) [inline charts/apisix/README.md:79]: Could you add the hyperlinks for the Kubernetes Volume and VolumeMount?

Comment (anjia0532) [reply charts/apisix/README.md:79]: Added it to readme.md or values.yaml ?

Comment (tokers) [reply charts/apisix/README.md:79]: README is better.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 .DS_Store
 test_**.yaml
+.idea
```
Comment (tao12345666333) [inline .gitignore:3]: Was this added accidentally?

File: charts/apisix/README.md (hunk 1 above)
Comment (anjia0532) [reply charts/apisix/README.md]: Done

File: .gitignore (hunk 2 above)
Comment (anjia0532) [reply .gitignore:3]: Idea and vscode is popular ide, `.idea` is jetbrains idea's project dir, Do I need to remove of it?

Comment (anjia0532) [reply .gitignore]: OK,removed

File: charts/apisix/README.md [hunk 3]
```diff
@@ -76,6 +76,8 @@ The following tables lists the configurable parameters of the apisix chart and t
 | `apisix.luaModuleHook.configMapRef.name` | Name of the ConfigMap where the lua module codes store | "" |
 | `apisix.luaModuleHook.configMapRef.mounts[].key` | Name of the ConfigMap key, for setting the mapping relationship between ConfigMap key and the lua module code path. | `""` |
 | `apisix.luaModuleHook.configMapRef.mounts[].path` | Filepath of the plugin code, for setting the mapping relationship between ConfigMap key and the lua module code path. | `""` |
+| `extraVolumes` | Additional `volume`, [k8s docs volumes](https://kubernetes.io/docs/concepts/storage/volumes/). | `[]` |
+| `extraVolumeMounts` | Additional `volumeMounts`, [k8s docs volumes](https://kubernetes.io/docs/concepts/storage/volumes/). | `[]` |
```
Comment (tokers) [inline charts/apisix/README.md:80]:
```suggestion
| `extraVolumeMounts` | Additional `volumeMounts`, See [Kubernetes Volumes](https://kubernetes.io/docs/concepts/storage/volumes/) for the detail. | `[]` |
```

File: charts/apisix/README.md [hunk 4]
```diff
@@ -76,6 +76,8 @@ The following tables lists the configurable parameters of the apisix chart and t
 | `apisix.luaModuleHook.configMapRef.name` | Name of the ConfigMap where the lua module codes store | "" |
 | `apisix.luaModuleHook.configMapRef.mounts[].key` | Name of the ConfigMap key, for setting the mapping relationship between ConfigMap key and the lua module code path. | `""` |
 | `apisix.luaModuleHook.configMapRef.mounts[].path` | Filepath of the plugin code, for setting the mapping relationship between ConfigMap key and the lua module code path. | `""` |
+| `extraVolumes` | Additional `volume`, [k8s docs volumes](https://kubernetes.io/docs/concepts/storage/volumes/). | `[]` |
```
Comment (tokers) [inline charts/apisix/README.md:79]:
```suggestion
| `extraVolumes` | Additional `volume`, See [Kubernetes Volumes](https://kubernetes.io/docs/concepts/storage/volumes/) for the detail. | `[]` |
```

**Review by tokers — APPROVED** [review]

**Review by tao12345666333 — APPROVED** [review]
LGTM

**Pull request merged**